src/frontend/shared-components/src/footer/Footer.js

```diff
@@ -27,7 +28,15 @@ export const FooterToggler = () => (
 export const Footer = () => (
   <footer className="footer">
     <nav className="navbar navbar-expand-sm navbar-dark justify-content-end">
-      <FooterToggler />
+      {/* <FooterToggler /> */}
```

Comment:
is this commented out? if so, is it required?

Reply:
I had done that for testing, should be fixed now

Reply:
toggle on, my wayward son.